tcf_website/views/team_alums.json

```diff
@@ -12,18 +12,59 @@
     }
   ],
   "contributors": [
+    {
+      "group_name": "Class of 2022",
+      "members": [
+                  {"name": "Harshita Pathipati",
+                  "role": "Developer",
+                  "class": "2022",
+                  "img_filename": "hpathipati.jpg",
+                  "github": ""
+                  }
+                 ]
+    },
+    {
+      "group_name": "Class of 2021",
+      "members": [
+                  {"name": "Brian Yu",
+                   "role": "19-20 President",
```

Comment:
would be more consistent with the other alum profiles to take out the "19-20"

Reply:
Done